### DISCUSSION TITLE
[FEATURE-REQUEST] En la sección Reports cómo se sincronizan campos nuevos de un formulario

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#280] [3. Ideas & Feature Requests] [answered]
Hola

Tengo dos dudas:

**1.- Se debe sincronizar las tablas cuando agrego un campo adicional a un survey?**

**Fundamento**: Resulta que en un survey agregué dos campos (uno de propiedad y uno de texto) y no aparecen en el campo de respuestas.
![image](https://user-images.githubusercontent.com/61419471/122578512-f05c2180-d021-11eb-927a-358910703922.png)

**2.- Al descargar un reports se descarga con los displays de los campos?**

**Fundamento**: Descargué un report de survey y me aparecen con el texto "Display en los campos" y deberá aparecer los nombres de cada campo. Sale el id en campo y el otro el nombre de ese id

![image](https://user-images.githubusercontent.com/61419471/122578781-3a450780-d022-11eb-8ab2-708220571609.png)

Gracias :>

**Comment by NicolasPLG** [comment]
Respondo la pregunta 1

**Respuesta 1**: Si se debe botar la tabla para que se reflejen los datos adicionales. gracias @jdurandpaz 

Con respecto a la pregunta 2 es un feature request?

gracias

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG gracias a @jdurandpaz por responder la primera pregunta, efectivamente hay que recrear la tabla.

Respecto a lo segundo, efectivamente es un feature request, por lo que si deseas priorizarlo, contacta a tu representante Cotalker.
